lxd/container_test.go

```diff
@@ -48,7 +48,7 @@ func (suite *containerTestSuite) TestContainer_ProfilesMulti() {
 			Name:        "unprivileged",
 			Description: "unprivileged",
 			Config:      map[string]string{"security.privileged": "true"},
-			Devices:     config.Devices{},
+			Devices:     map[string]map[string]string{},
```

Comment:
Same as the other commit, I don't see why we need to set this thing in the first place, can you try just removing it?